doc/rustdocfx/generate.go

```diff
@@ -320,22 +305,6 @@ func processImplementation(c *crate, id string, page *docfxManagedReference, par
 	return nil
 }
 
-func newDocfxItemFromAssocConst(c *crate, parent *docfxItem, id string) (*docfxItem, error) {
```

Comment:
I take it these removed functions are unused code?

cc: @PhongChuong to add them back if they are needed.